environment.yml

```diff
@@ -5,7 +5,7 @@ channels:
 dependencies:
   - astropy
   - black
-  - echemdb>=0.4.0,<0.5.0
+  #- echemdb>=0.4.0,<0.5.0
```

Comment:
we have to pip install an unmerged echemdb PR due to circular dependencies. The install is a pip install below.